src/ixdat/techniques/ms.py

```diff
@@ -512,8 +513,7 @@ def update_l_cap(self, gases=[]):
 
         return self.l_cap
 
-    def calc_n_dot_0(
-        self, gas=None, w_cap=None, h_cap=None, l_cap=None, T=None, p=None):
+    def calc_n_dot_0(self, gas=None, w_cap=None, h_cap=None, l_cap=None, T=None, p=None):
```

Comment:
Looks good. It's got both the new method above and this method definition black-formatted.